packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -11,15 +11,38 @@ import { Divider, Timestamp } from '@patternfly/react-core';
 
 ## PatternFly 6.1.1
 
+### Promoted package versions
+- [@patternfly/data-view6.1.0](https://www.npmjs.com/package/@patternfly/react-data-view)
+
 ### Extension updates 
 
 As a quick follow up to our last release, this update publishes the new data view extension.
 
+<Divider />
+
 <Timestamp>December 2024</Timestamp>
 
 ## PatternFly 6.1
 
 ### Promoted package versions
+- patternfly/patternfly ([changelog](https://github.com/patternfly/patternfly/releases/tag/v6.1.0))
+    - [@patternfly/patternfly6.1.0](https://www.npmjs.com/package/@patternfly/patternfly)
```

Comment:
I think we should be consistent about having the @, two lines have it and the rest don't, and maybe adding a space or hyphen between the package and the version number?

`@patternfly/patternfly@6.1.0` would follow a pattern that I've seen notated in `package.json` before, but I'm not sure there's a convention for specifying a specific version.

Reply:
Oops, good callout! Totally dropped the @ as I was building the list. 

I'll make updates with your suggestions, tysm!